examples/CustomResizeExample.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+"use strict";
+
+import FakeObjectDataListStore from './helpers/FakeObjectDataListStore';
+import { TextCell } from './helpers/cells';
+import { Table, Column, Plugins } from 'fixed-data-table-2';
+import React from 'react';
+
+
+class ResizeExample extends React.Component {
```

Comment:
Instead of adding a new example, let's just modify the existing example.

Reply:
Done.. Added a todo